src/wallet/sqlite.cpp

```diff
@@ -18,6 +18,7 @@
 #include <stdint.h>
 
 static const char* DATABASE_FILENAME = "wallet.dat";
+static const int32_t WALLET_SCHEMA_VERSION = 0;
```

Comment:
10ceb039e8361d91a9169da8f68872ad4b564403
```suggestion
static constexpr int32_t WALLET_SCHEMA_VERSION = 0;
```

Reply:
Done